src/services/expenses.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+let nextExpenseId = 1;
```

Comment:
I think, more reliable in this case would  use Math.random multiply with new Date inside of the function where you use it.